tests/integration/ceph_mgr_test.go

```diff
@@ -122,19 +114,73 @@ func (s *CephMgrSuite) execute(command []string) (error, string) {
 	return s.installer.Execute("ceph", orchCommand, s.namespace)
 }
 
+func (s *CephMgrSuite) enableOrchestratorModule() error {
+	logger.Info("Enabling Rook orchestrator module: <ceph mgr module enable rook --force>")
+	err, output := s.installer.Execute("ceph", []string{"mgr", "module", "enable", "rook", "--force"}, s.namespace)
+	logger.Infof("output: %s", output)
+	if err != nil {
+		logger.Infof("Not possible to enable rook orchestrator module: %q", err)
+		return err
+	}
+	logger.Info("Setting orchestrator backend to Rook .... <ceph orch set backend rook>")
+	err, output = s.execute([]string{"set", "backend", "rook"})
+	logger.Infof("output: %s", output)
+	if err != nil {
+		logger.Infof("Not possible to set rook as backend orchestrator module: %q", err)
+	}
+	return err
+}
+
 func (s *CephMgrSuite) waitForOrchestrationModule() {
 	var err error
+
+	// Status struct
+	type orchStatus struct {
+		Available bool   `json:"available"`
+		Backend   string `json:"backend"`
+	}
+
+	err = s.enableOrchestratorModule()
```

Comment:
Since we expect the rook module to already be enabled by the operator, how about if we remove this call that is before the `for` loop?